packages/salesforcedx-vscode-core/test/sfdx-simple/force-app/main/default/aura/demoApp/demoApp.app

```diff
@@ -0,0 +1,3 @@
+<aura:application>
```

Comment:
I will get rid of it. I looked to see why we had a demo page or demo class in it but couldn't find any references to it in the test so I followed suit, assuming we wanted a demo of each type being in sfdx-simple.